### DISCUSSION TITLE
Interactive diagram/images

### DISCUSSION THREAD

**Discussion by ismaelrumzan** [remotion-dev/remotion#1246] [Q&A]
Hi
I have a series of frames, which will be rendered as React components containing images and for each images there will be a click action on an area
Then, the series will be rendered by remotion.
However, I would like each frame to stop and go on to the next frame based on the click action.
Is it possible to get this done?

**Comment by JonnyBurger** [comment]
Hi @ismaelrumzan!
Generally, it is possible. You can hook up click handlers to the events of the player: https://remotion.dev/docs/player/api

We don't optimize for this usecase though, our main usecase is rendering videos independent from user interactions.
See also this project by a friend of ours: https://liqvidjs.org/
